packages/blueprint-component-mapper/src/files/checkbox.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import MultipleChoiceList from '@data-driven-forms/common/src/multiple-choice-list';
+import { Checkbox, FormGroup, Intent } from '@blueprintjs/core';
+
+import FormGroupWrapper from './form-group';
+import propsCatcher from '../common/props-catcher';
+import BlueprintContext from './blueprint-context';
+
+const SingleCheckbox = ({ input, isDisabled, label, providerRequired, ...props }) => (
+  <Checkbox
+    label={label}
+    {...propsCatcher(props)}
+    {...(props.isRequired && {
+      label: (
+        <span>
+          {label} {providerRequired}
+        </span>
+      )
+    })}
+    {...(isDisabled && { disabled: true })}
```

Comment:
it probably is pointless :D

Reply:
It isn't.. the single checkbox is used in two contexts 

1) standalone
2) in multiple checkboxes

In standalone it receives `disabled` prop from the wrapper, in multiple it receives `isDisabled`...

however, it can be changed to `disabled={disabled || isDisabled}`